main.cpp

```diff
@@ -52,85 +52,85 @@ arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:       remote
+//  action 1:   changes the channel
+remote.changeChannel();
+//  action 2:   turns up the volume
+remote.increaseVolume();
+//  action 3:   opens the menu
+remote.openMenu();
 //  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:       cat
+//  action 1:   begs for food
+cat.begForFood();
+//  action 2:   meow
+cat.meow();
+//  action 3:   kneads
+cat.knead();
 //  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:       guitar pedal
+//  action 1:   distorts
+pedal.distort();
+//  action 2:   creates delay
+pedal.addDelay();
+//  action 3:   add a lil flange, why not
+pedal.Flanger();
 //  4)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:       shoe
+//  action 1:   protects your feet
+shoe.protectFeet();
+//  action 2:   loosens the laces
+shoe.loosenLaces();
+//  action 3:   lets in water even though it was called "waterproof" in the store
+shoe.letInWater();
 //  5)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:       smart watch
+//  action 1: tells the time
+watch.tellTime();
+//  action 2:   displays messages
+watch.displayMessage();
+//  action 3:   counts your steps
+watch.countSteps();
 //  6)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:       towel
+//  action 1:   drys you off
+towel.dryUser();
+//  action 2: absorbs sound in the bathroom
+towel.absordSound();
+//  action 3: retains a smell
+towel.retainSmell();
 //  7)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:       smoke alarm
+//  action 1:   sniffs out your good cooking
+alarm.detectSmoke();
+//  action 2:   beeps like mad
+alarm.beep();
+//  action 3:   uses up the battery
+alarm.depleteBattery();
 //  8)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:       oven
+//  action 1:   preheats
+oven.preHeat();
+//  action 2:   warms up the room
+oven.warmRoom();
+//  action 3:   cooks your food
+oven.cookFood();

```

Comment:
As explained in the instructions, here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
REPLACE the highlighted noun/action as follows:
- [ ] come up with a new noun and 3 actions
- [ ] pick a noun that needs `camelCase` for the name
- [ ] pick actions that need `camelCase` for their name.

Do not create an 11th noun.

Once you have completed the above-requested changes, please do the following:
- click each checkbox to mark each requested change as completed.
- click the `[Resolve Conversation]` button.
- Re-request a review of your pull request. It's on the right side of the Pull request page where you originally requested my review.
- Send me the link to your pull request in a new message.